gammacat/utils.py

```diff
@@ -84,6 +84,17 @@ def fill_list(cls, data, key):
             return cls.fill_value['string']
 
 
+def convert_source_id(source):
```

Comment:
Zero-padding can be done with one line -> remove this helper function.
https://pyformat.info/#number
```
'{:04d}'.format(42)
```